service/src/main/java/uk/gov/hmcts/reform/fpl/utils/BigDecimalHelper.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+@Slf4j
+public class BigDecimalHelper {
+
+    private static final BigDecimal HUNDRED = BigDecimal.valueOf(100);
+
+    private BigDecimalHelper() {
+
+    }
+
+    public static String toCCDMoneyGBP(BigDecimal amount) {
+        if (amount == null) {
+            return "";
+        }
+
+        BigDecimal pence = amount.multiply(HUNDRED);
+        return String.valueOf(pence.longValue());
+    }
+
+    public static Optional<BigDecimal> fromCCDMoneyGBP(String amount) {
+        try {
+            int integer = Integer.parseInt(amount);
+            return Optional.of(BigDecimal.valueOf(integer, 2));
```

Comment:
What's the reason we don't use `BigDecimal` constructor?

Reply:
`valueOf` and the constructor do slightly different things, `valueOf` uses the string representation of the passed value where as the constructor uses the passed primitive as accurately as possible.
See [javadoc](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/math/BigDecimal.html#%3Cinit%3E(double)) and [stack overflow post](https://stackoverflow.com/a/7186298)

Reply:
Thanks for explanation